gravitee-apim-gateway/gravitee-apim-gateway-policy/src/main/java/io/gravitee/gateway/reactive/policy/ConditionalPolicy.java

```diff
@@ -29,16 +29,20 @@
  * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
  * @author GraviteeSource Team
  */
-public class ConditionalPolicy implements Policy, ConditionSupplier {
+public class ConditionalPolicy implements HttpPolicy, ConditionSupplier {
```

Comment:
Should we rename this HttpConditionalPolicy to be consistent ?
```suggestion
public class HttpConditionalPolicy implements HttpPolicy, ConditionSupplier {
```